modules/backendscheduler/provider/metrics.go

```diff
@@ -16,4 +16,14 @@ var (
 		Name:      "compaction_tenant_reset_total",
 		Help:      "The number of times the tenant is changed",
 	}, []string{"tenant"})
+	metricTenantBackoff = promauto.NewCounter(prometheus.CounterOpts{
+		Namespace: "tempo_backend_scheduler",
+		Name:      "compaction_tenant_backoff_total",
+		Help:      "The number of times the backoff is triggered",
+	})
+	metricTenantNilJob = promauto.NewCounter(prometheus.CounterOpts{
+		Namespace: "tempo_backend_scheduler",
+		Name:      "compaction_tenant_nil_job_total",
```

Comment:
let's rename this to something like `compaction_tenant_backoff_job_total` or something that indicates that we have no jobs and workers are not getting jobs because we have no work to handout.

Reply:
maybe call it NoJobs instead of backoff, because NoJob is more clear??

Reply:
Good callout.  I added a commit to rename it to `empty` which I think reads true enough.